modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -35,29 +37,83 @@ definition {
 	}
 
 	@description = "LPS-178192. Verifies only if the setting has never been changed by the user, either when signed in or not signed in, will the default value be displayed.."
-	@ignore = "Test Stub"
 	@priority = 5
 	test BrowserLocalStorageSmoke {
 		property portal.acceptance = "true";
 
-		//task ("Given When Then") {
+		task ("Given created new user A") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfnA",
+				userLastName = "userlnA",
+				userScreenName = "usersnA");
+		}
 
-		// // Given a new created user A
-		// // And signed out as guest user
-		// // When toggle  show underline to OFF state
-		// // And sign in as a new user for first time
-		// // And toggle show under to ON
-		// // And user signs out
-		// // Then
-		// // show underline is set to OFF
-		// // And When
-		// // guest user toggles show underline to ON
-		// // And guest user toggles show underline to OFF
-		// // And sign in as user A
-		// // Then show underline is set to ON state
+		task ("And signed out as guest user") {
+			SignOut.signOut();
+		}
 
-		//}
+		task ("When the toggle shows underline to ON state") {
+			AccessibilityMenu.openAccessibilityMenu();
+
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+			AssertChecked.assertCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
+
+		task ("And sign in as a new user for first time") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfnA userlnA");
+		}
+
+		task ("And the toggle shows under to OFF") {
+			AccessibilityMenu.openAccessibilityMenu();
+
+			Uncheck.uncheckToggleSwitch(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
+
+		task ("And user signs out") {
+			SignOut.signOut();
+		}
+
+		task ("Then the toggle shows underline is set to ON") {
+			AccessibilityMenu.openAccessibilityMenu();
+
+			AssertChecked.assertCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
+
+		task ("And When guest user toggles show underline to ON") {
+			Navigator.openURL();
+
+			AccessibilityMenu.openAccessibilityMenu();
+
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
+
+		task ("And sign in as user A") {
+			User.logoutAndLoginPG(
```

Comment:
Missing a step here. Please add in `And guest user toggles show underline to ON`

		// // Then show underline is set to ON
		// // And When guest user toggles show underline to OFF
		**// // And guest user toggles show underline to ON**
		// // And sign in as user A

Reply:
yes

Reply:
Hi @eltonccdantas , please include both steps:

**And When guest user toggles show underline to OFF
And guest user toggles show underline to ON**